Swerve/SwerveModules/SwerveModuleIO.java

```diff
@@ -0,0 +1,106 @@
+package POPLib.Swerve.SwerveModules;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.littletonrobotics.junction.AutoLog;
+import java.lang.Cloneable;
+import java.lang.Override;
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+
+public interface SwerveModuleIO {
+    @AutoLog
+    public static class ModuleIOInputsAutoLogged implements LoggableInputs, Cloneable{
+        public boolean driveConnected = false;
+        public double drivePositionRad = 0.0;
+        public double driveVelocityRadPerSec = 0.0;
+        public double driveAppliedVolts = 0.0;
```

Comment:
Don't know if we ever apply a voltage to drive motors other then when running sysid would remove

Reply:
@kchatre you didn't remove it 😲

bro was so close to getting his pr approved today too.